README.md

```diff
@@ -235,10 +235,10 @@ changes to the console.  To use it, simply decorate your reducer with the
 default export of the `innerself/logger` module:
 
 ```javascript
-import { create_store } from "innerself";
+import { createStore } from "innerself";
 import with_logger from "innerself/logger";
```

Comment:
For consistency, let's rename this to `withLogger`, please.

Reply:
Alright. I'll push another commit

Reply:
Changed `with_logger` to `withLogger`